sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebViewProvider.java

```diff
@@ -16,7 +13,10 @@ interface HCaptchaWebViewProvider extends
         OnOpenListener,
         OnSuccessListener<HCaptchaTokenResponse>,
         OnFailureListener {
-    @Nullable
-    WebView getWebView();
```

Comment:
Probably make sense to rename `HCaptchaWebViewProvider` to something else because it doesn't provide views anymore

Reply:
As you proposed in dm, `IHCaptchaVerifier` sound good